samples/UDPEchoServ4.js

```diff
@@ -0,0 +1,14 @@
+var PORT = 33333;
```

Comment:
I see that other examples have copyright... Well, I'm personally kinda against putting copyright on trivial examples, that may only complicate users' lives - either make them bother to deal with process regarding copyrights, or make them violators. But if there's a policy that every file in the project should have a copyright notice, please re-confirm, and I'll add it.